src/randoop/sequence/AbstractGenerator.java

```diff
@@ -113,41 +117,50 @@
+   * The list of failure sequences to be saved as JUnit tests
+   */
+  public List<ExecutableSequence> outFailureSeqs = new ArrayList<>();
+
+  /**
+   * The list of regression sequences to be save as JUnit tests
+   */
+  public List<ExecutableSequence> outRegressionSeqs = new ArrayList<>();
+  
+  /**
+   * A filter to determine whether a sequence should be added to the final 
+   * sequence list outSeqs.
    */
-  public List<ExecutableSequence> outSeqs = new ArrayList<ExecutableSequence>();
+  public Predicate<ExecutableSequence> outputTest;
 
   /**
-   * A list of filters that can be installed to help determine if a sequence
-   * should be added to the final sequence list outSeqs.
+   * Visitor to generate checks for a sequence.
    */
-  public List<ITestFilter> outputTestFilters;
+  protected TestCheckGenerator checkGenerator;
   
- 
   /**
    * Constructs a generator with the given parameters.
    * 
    * @param operations Statements (e.g. methods and constructors) used to create sequences. Cannot be null.
    * @param timeMillis maximum time to spend in generation. Must be non-negative.
-   * @param maxSequences maximum number of sequences to generate. Must be non-negative.
-   * @param componentManager component manager to use to store sequences during component-based generation.
+   * @param maxGenSequences  the maximum number of sequences to generate. Must be non-negative.
+   * @param maxOutSequences  the maximum number of sequences to output. Must be non-negative.
+   * @param componentManager  the component manager to use to store sequences during component-based generation.
    *        Can be null, in which case the generator's component manager is initialized as <code>new ComponentManager()</code>.
    * @param stopper Optional, additional stopping criterion for the generator. Can be null.
    * @param listenerManager Manager that stores and calls any listeners to use during generation. Can be null.
-   * @param testfilters List of filters to determine which sequences to output. Can be null or empty.
    */
-  public AbstractGenerator(List<Operation> operations, long timeMillis, int maxSequences, ComponentManager componentManager,
-      IStopper stopper, RandoopListenerManager listenerManager, List<ITestFilter> testfilters) {
+  public AbstractGenerator(List<Operation> operations, long timeMillis, 
+      int maxGenSequences, int maxOutSequences, ComponentManager componentManager,
+      IStopper stopper, RandoopListenerManager listenerManager) {
     assert operations != null;
 
     this.maxTimeMillis = timeMillis;
-
-    this.maxSequences = maxSequences;
-
+    this.maxGeneratedSequences = maxGenSequences;
+    this.maxOutputSequences = maxOutSequences;
     this.operations = operations;
-
-    this.executionVisitor = new MultiVisitor();
-
+    this.executionVisitor = new DummyVisitor();
+    this.outputTest = new AlwaysFalse<>();
+    
```

Comment:
Your editor seems to be leaving spaces on blank lines.  Can you configure it not to do so?